e2e/test_operator_upgrades/operator_upgrades_test.go

```diff
@@ -376,6 +379,22 @@ var _ = Describe("Operator Upgrades", Label("e2e"), func() {
 			)
 			Expect(err).NotTo(HaveOccurred())
 
+			// Wait for the server process to restart.
+			time.Sleep(40 * time.Second)
+
+			// Check if the restarted process is showing up in IncompatibleConnections list in status output.
+			status := fdbCluster.GetStatus()
+			log.Println("IncompatibleProcesses:", status.Cluster.IncompatibleConnections)
+
+			// NOTE: The restarted process doesn't show up in IncompatibleConnections list consistently, hence
```

Comment:
Could you open an issue for that? We should investigate this issue on FDB side and make sure we understand why the connection is not appearing in the incompatible connection list consistently. And does `The restarted process doesn't show up in IncompatibleConnections list consistently` mean that 1.) It's not appearing at all in the list? 2.) It's in a state where the address is appearing and disappearing? If I remember correct the detection time was something around 100 seconds, but I would need to review the FDB code. again.

Reply:
Let me try changing "time.Sleep()" to 100 seconds and see what happens.